ui/app/AppLayouts/Wallet/controls/ManageTokensCommunityTag.qml

```diff
@@ -27,6 +32,7 @@ Control {
 
     QtObject {
         id: d
+        property bool unknown: !!root.communityName ? root.communityName.startsWith("0x") : false
```

Comment:
`readonly`

Reply:
```suggestion
        redaonly property bool unknownCommunityName: !!root.communityName ? root.communityName.startsWith("0x") : false
```

Reply:
Seems like we allow Community names starting with 0x 
![image](https://github.com/status-im/status-desktop/assets/11161531/60a539a8-cef1-4012-8189-665efb344fc2)